test/facet_fields.csv

```diff
@@ -3,9 +3,9 @@
 // readable_field_name: Field will have this name in Elasticsearch index and
```

Comment:
Oops, I meant to delete this directory in #81. I'll send a PR to delete.

Reply:
Thanks